halo/network/network_id.hpp

```diff
@@ -30,3 +31,22 @@ int32_t get_network_id_from_object(MemRef object); // returns -1 if not found.
 
 // Initialize all the pointers.
 void init_network_id();
+
+struct SyncedObjectHeader{
+    uint32_t max_count;
+    uint32_t int1;                  //0x4
+    uint32_t int2;                  //0x8
```

Comment:
Do we not know what these `int1`, `int2`, etc... values are yet? If that's the case, there should be a comment above this noting that.